iceoryx_posh/include/iceoryx_posh/internal/popo/sample.inl

```diff
@@ -96,6 +109,16 @@ inline void Sample<T>::release() noexcept
     m_samplePtr.release();
 }
 
+template <typename T>
+template <typename... Args>
+inline void Sample<T>::emplace(Args&&... args) noexcept
```

Comment:
We should probably get rid of this, I left it here but there is currently no way to obtain an uninitialized sample as far as I am aware. What do you think @elBoberido

Reply:
yes, remove it. It not necessary since loan now takes variadic arguments